popgen/nkpCacheIntegral.go

```diff
@@ -0,0 +1,34 @@
+package popgen
+
+import (
+	"github.com/vertgenlab/gonomics/numbers"
+	"math"
+)
+
+func LogIntegrateStationarityCache(alpha float64, n int, k int, stepSize int, alleleFrequencyCache []float64, nkpCache [][][]float64) float64 {
+	var bins int = len(nkpCache[n][k])-1
+	var deltaX float64 = (RightBound - LeftBound) / float64(bins)
```

Comment:
This should have "stepSize" as part of the calculation to determine deltaX.  This bug might actually give you a significant speedup because right now I bet that every time through the loop of how many bins to use, it gets a really different answer.  This may cause it to always use 100,000 bins.

Reply:
Great call! Should have stepSize as a multiplier, as a stepsize of 10 has 10 times the deltaX, right?